documentation/asciidoc/topics/proc_defining_environment_var.adoc

```diff
@@ -0,0 +1,51 @@
+[id='defining-environment-variables_{context}']
+= Defining environment variables in the {ispn_operator} subscription
+
+[role="_abstract"]
+Define environment variables in your {ispn_operator} subscription that automatically propagate to each {brandname} po and service managed by the corresponding Operator.
+
+`spec.config.env` field is an array of environment variable definitions.
+You can add one or more environment variable names and corresponding values.
+
+.Prerequisites
+* Ensure the Operator Lifecycle Manager (OLM) is installed.
+
+.Procedure
+
+Define environment variables in your {ispn_operator} subscription using the `spec.config.env` field:
+
+. Use the `{oc_edit} subscription` command.
```

Comment:
I think it would be better if we show creating a `Subscription` with the variables instead of editing an existing one. So `oc apply` instead of  `oc edit`. The example yaml will need to be updated to include the rest of the Subscription fields.

Reply:
I see that we have edit for the console as well. Maybe it's best to have an apply workflow for the CLI and edit for the console? Or have edit for both and just apply for CLI, WDYT?

Reply:
Thanks for the comments! I agree that we should add apply workflow. My proposal is to have a follow up step showing how to edit the existing subscription. I'll try and see how it looks